app/models/ems_event.rb

```diff
@@ -10,6 +11,24 @@ class EmsEvent < EventStream
     'Rename_Task',
   ]
 
+  def self.description
+    "Management Events"
+  end
+
+  def self.group_names_and_levels
+    event_groups.each_with_object({}) do |arr, hsh|
```

Comment:
I'm not sure what the array is here, but can we deconstruct it here and give the parameters names?

Reply:
Something like:

```suggestion
    event_groups.each_with_object({}) do |(group_name, group_details), hsh|
```

Reply:
Good idea.